lib/Devel/TraceUse.pm

```diff
@@ -92,9 +92,11 @@ sub trace_use
     };
 
     # info about the loading module
-    # (our require override adds one frame)
     my $caller = $info->{caller} = {};
-    @{$caller}{@caller_info} = caller(1);
+    my $caller_initial_level = 1;  # one for the require() wrapper
+    $caller_initial_level++ if $^V >= 5.37.7; # and another for modern perls
```

Comment:
This code does not work on 5.8.9. Before Perl v5.10.0 $^V was represented as a v-string

Reply:
fixed in the latest push.